docs.html

```diff
@@ -84,16 +84,13 @@
 <h2>
 <a name="releasing" class="anchor" href="#releasing"><span class="octicon octicon-link"></span></a>Releasing a module</h2>
 
-<p>If modulesync is enabled for the module, run it to ensure dotifles are up to date.</p>
-<p>Create a 'release pr'. This is a PR that modifies the changelog, bumps the version name, etc. Ask for reviews on it. This is where the community can agree or disagree with the version number you have chosen, and otherwise vote on whether to release or not. Example: https://github.com/puppet-community/puppet-archive/pull/98 </p>
+<p>run modulesync to ensure dotfiles are up to date.</p>
+<p>Create a 'release pr'. This is a PR updates the changelog, and bumps the version number. Ask for reviews on it. This is where the community can agree or disagree with the version number you have chosen, and otherwise vote on whether to release or not. Here's an example: <a href="https://github.com/puppet-community/puppet-extlib/pull/43">puppet-extlib's 0.10.7 release</a></p>
 <p>Get community feedback on the release pr, get it merged.</p>
 <p>Checkout master</p>
-<p>Create a signed tag at the version:  git tag -s 1.0.2</p>
-<p>For the text of the signed tag use: "Release 1.0.2"  </p>
-<p>Push the signed tag to github: git push --tags origin master </p>
+<p>run `bundle exec rake travis_release`</p>
```

Comment:
If necessary, run `bundle install` before continuing.
Run the rake target `travis_release`. This will create a new tag using the current version, bump the current version to the next PATCH version and add '-rc0' to the end, commit the change and push it to origin: `bundle exec rake travis_release`